.github/workflows/rethnet-ci.yml

```diff
@@ -6,13 +6,15 @@ on:
       - $default-branch
       - "rethnet/main"
     paths:
+      - ".github/workflows/rethnet-ci.yml"
       - "config/**"
       - "crates/**"
       - "Cargo.toml"
       - "rust-toolchain"
   pull_request:
     branches: ["**"]
     paths:
+      - ".github/workflows/rethnet-ci.yml"
```

Comment:
Wait, is this necessary? I mean, it does make sense. Maybe we should do it in the other workflows too.

Reply:
Yes, otherwise it would not re-run the CI for rethnet-ci